src/setupWorker/start/utils/activateMocking.ts

```diff
@@ -3,15 +3,21 @@ import { StartOptions, SetupWorkerInternalContext } from '../../glossary'
 export const activateMocking = async (
   context: SetupWorkerInternalContext,
   options?: StartOptions,
+  shared = false,
```

Comment:
This option should be accessed from the second `options` argument already passed here.